package/MDAnalysis/units.py

```diff
@@ -239,6 +239,7 @@
     'ps': 1.0, 'picosecond': 1.0,  # 1/1.0
     'fs': 1e3, 'femtosecond': 1e3,  # 1/1e-3,
     'ns': 1e-3, 'nanosecond': 1e-3,  # 1/1e3,
+    'ms': 1e-9, 'millisecond': 1e-9,  # 1/1e9,
```

Comment:
Since you reach this far, why don't you add also microsecond? :relaxed: 

Good job! :+1:

Reply:
On it. Thanks for the review!